src/dag-node/sortLinks.js

```diff
@@ -8,12 +8,23 @@ const linkSort = (a, b) => {
 }
 
 /**
- *
+ * Returns new sorted links array.
  * @param {Array} links
  * @returns {Array}
  */
 const sortLinks = (links) => {
```

Comment:
If we're switching to in-place sorting, is this function used somewhere else in the module or can it be removed?

Reply:
Only reason I have not done it is I do not know if anything outside of the package might be making use of this. If we're comfortable removing it, I'm happy to do so

Reply:
Made a change that makes replaces existing `sortLinks` with mutable sortLinks`. One thing that makes me bit uncomfortable about this change is that it's default import and change will in behavior will not be immediately obvious (if it was named, I'd just renamed the function).